engine/src/render/tone_mapping/plugin.cpp

```diff
@@ -29,17 +40,25 @@ namespace
         ShaderBindingPoint gammaBP;
         ShaderBindingPoint exposureBP;
         ShaderBindingPoint hdrBP;
+        ShaderBindingPoint screenSizeBP;
+        ShaderBindingPoint fxaaConfigBP;
+
         VertexArray screenQuad;
 
+        ConstantBuffer fxaaConfigCB;
+
         State(RenderDevice& renderDevice, const ShaderPipeline& pipeline)
             : pipeline(pipeline)
         {
             hdrBP = pipeline->getBindingPoint("hdrTexture");
             gammaBP = pipeline->getBindingPoint("gamma");
             exposureBP = pipeline->getBindingPoint("exposure");
-            CUBOS_ASSERT(hdrBP && gammaBP && exposureBP, "hdrTexture, gamma and exposure binding points must exist");
-
+            screenSizeBP = pipeline->getBindingPoint("screenSize");
+            fxaaConfigBP = pipeline->getBindingPoint("FxaaConfig");
+            CUBOS_ASSERT(hdrBP && gammaBP && exposureBP && screenSizeBP && fxaaConfigBP,
+                         "hdrTexture, gamma, exposure, screeSize and fxaaConfig binding points must exist");
             generateScreenQuad(renderDevice, pipeline, screenQuad);
+            fxaaConfigCB = renderDevice.createConstantBuffer(sizeof(FxaaConfig), nullptr, Usage::Default);
```

Comment:
When I tested on my machine, even though it worked, it complained about using `glBufferSubData` to update a static buffer. Changing it to dynamic fixed the error.
```suggestion
            fxaaConfigCB = renderDevice.createConstantBuffer(sizeof(FxaaConfig), nullptr, Usage::Dynamic);
```